app/tests.py

```diff
@@ -255,12 +282,12 @@ def test_spotify_parser_valid_playlist_id(self):
 
     def test_spotify_parser_invalid_playlist_id(self):
         playlist_id = "thisdoesnotexist"
-        with self.assertRaises(SpotifyWebsiteParserException):
+        with self.assertRaises(SpotifyInvalidUrlException):
```

Comment:
Does this work for you? In my case a SpotifyWebsiteParserException is raised like before

Reply:
Oh, it's probably cos I added backwards compatibility later on. Strange though, I could have sworn all tests passed prior to submitting. Will fix.